### ISSUE TITLE
great work, elastic?

### ISSUE THREAD

**Issue by andrewarrow** [balzaczyy/golucene#10]
thanks for making this! I was about to start writing my own port to go cuz I just can't take java anymore. Any info on a golang elastic to query this?

**Comment by ironsweet** [comment]
Hi Andrew,

Glad you like it. This is not elasticseach port, btw, assuming you are
looking for es server port.

Andrew Arrow notifications@github.com于2016年9月13日 周二 12:08写道：

**Comment by andrewarrow** [comment]
there is: 
1) java lucene with java solr
2) java lucene with java elastic
3) golang golucene with gosolar
4) golang golucene with goelastic

i.e. do gosolar and goelastic exist yet? And if not has anyone pointed a normal java solr or java elastic to a golucene lucene index?

**Comment by ironsweet** [comment]
I don't think so. But as long as the index format is the same, it should not be a problem for sole or elastic search to consume the index, generated by other language port.

**Comment by janakact** [comment]
Maybe it's time great if someone can start such a project because, there are many benefits such as faster start time and low memory usage.
